end-to-end-tests/cypress/support/commands.js

```diff
@@ -112,3 +112,10 @@ Cypress.Commands.add("setTestSspJson", (sspJson) => {
   );
 });
 
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/
+Cypress.on('uncaught:exception', (err) => {
+    /* returning false here prevents Cypress from failing the test */
+    if (resizeObserverLoopErrRe.test(err.message)) {
+        return false
+    }
+})
```

Comment:
Why isn't this being resolved with the code that already exists?

https://github.com/EasyDynamics/oscal-editor-deployment/blob/a3bc3798b87220fec211554efdb443a558443874/end-to-end-tests/cypress/support/e2e.js#L22-L36

Reply:
Excellent question. I will look into that.